ECLIPs/ECLIP-2.md

```diff
@@ -0,0 +1,50 @@
+# Add OpenRPC Service Discovery
```

Comment:
```suggestion
# Add OpenRPC Service Discovery To JSON RPC Services
```

Reply:
JSON-RPC

Reply:
Or if we follow [StarIPs](https://github.com/BelfordZ/starIPs), it would be `ECIP-07be0c83`